lib/actionparameters.js

```diff
@@ -37,7 +37,7 @@ class ActionParameters {
         /**
          * Trimming the commit message because it is used as a param in uri of deployment api. And sometimes, it exceeds the max length of http URI.
          */
-        this._commitMessage = github.context.eventName === 'push' ? github.context.payload.head_commit.message.slice(0, 7000) : "";
+        this._commitMessage = github.context.eventName === 'push' ? github.context.payload.head_commit.message.slice(0, 2000) : "";
```

Comment:
Given @IGx89's point in issue https://github.com/Azure/webapps-deploy/issues/288 on URL encoding and how it will largely increase the length of the zipdeploy URL post-encode, should this trim be closer to 1000? Ditto for the other file change 👍